rov/movement/thruster/Force_Limiter.py

```diff
@@ -0,0 +1,117 @@
+from math import floor
+import operator
+
+class ForceLimiter(object):
+
+    # TODO: convertToPowerUnit()
+
+    def __init__(self):
+        self.a6vector = [0 for _ in range(6)]
+        self.an8vector = [0 for _ in range(8)]
+        self.last8vector = [0 for _ in range(8)]
+
+        self.mapper = Mapper()
+
+    """
+    1. acceptable_total_power
+    2. acceptable_individual_total
+    3. acceptable_total_change
+    4. acceptable_individual_change
```

Comment:
I think theses are the constants? I think it'd be good to explain what they mean (and there units if they have them), what the correspond to in `init_hw_constants` and why they're needed.

Reply:
they are not constants. They are the limiting functions. Constants are within them, but will be externalized to init_hw_constants when we finish figuring out what they need to be.

Reply:
Oh, derp. I see. Okay, well that's fine I guess.

Reply:
added "Order of Limiting Functions" comment